contracts/storage/market.sol

```diff
@@ -12,86 +12,81 @@ import "../shared/libraries/NumbersList.sol";
 import { ILoansEscrow } from "../escrow/escrow/ILoansEscrow.sol";
 import { ICollateralEscrow } from "../market/collateral/ICollateralEscrow.sol";
 import { ITToken } from "../lending/ttoken/ITToken.sol";
-/**
- * @notice Represents the terms of a loan based on the consensus of a LoanRequest
- * @param borrower The wallet address of the borrower
- * @param recipient The address where funds will be sent, only applicable in over collateralized loans
- * @param interestRate The consensus interest rate calculated based on all signer loan responses
- * @param collateralRatio The consensus ratio of collateral to loan amount calculated based on all signer loan responses
- * @param maxLoanAmount The consensus largest amount of tokens that can be taken out in the loan by the borrower, calculated based on all signer loan responses
- * @param duration The consensus length of loan time, calculated based on all signer loan responses
- */
+
 struct LoanTerms {
-    address payable borrower;
-    address recipient;
-    uint256 interestRate;
-    uint256 collateralRatio;
+    // Max size the loan max be taken out with
     uint256 maxLoanAmount;
-    uint256 duration;
+    // The timestamp at which the loan terms expire, after which if the loan is not yet active, cannot be taken out
+    uint32 termsExpiry;
 }
 
 enum LoanStatus { NonExistent, TermsSet, Active, Closed, Liquidated }
 
 struct Loan {
-    // The id of the loan for internal tracking
-    uint256 id;
+    // Account that owns the loan
+    address payable borrower;
     // The asset lent out for the loan
     address lendingToken;
```

Comment:
Would it be possible to have loanId's be made of the borrower's address and the number of loans they have? I feel like that would remove 2 SLOADs in most cases.

Reply:
not if you want to do something with the loan and only know the ID, there would be no way to know who the loan belongs to